docs/notebooks/08Calibrate_double_ended.ipynb

```diff
@@ -41,17 +41,18 @@
     "import warnings\n",
     "\n",
     "from dtscalibration import read_silixa_files\n",
-    "from dtscalibration.datastore_utils import (\n",
+    "from dtscalibration.dts_accessor_utils import (\n",
     "    suggest_cable_shift_double_ended,\n",
     "    shift_double_ended,\n",
     ")\n",
-    "from dtscalibration import DtsAccessor   # noqa: E401\n",
+    "\n",
+    "# The following line introduces the .dts accessor for xarray datasets\n",
+    "import dtscalibration  # noqa: E401  # noqa: E401\n",
     "from dtscalibration.variance_stokes import variance_stokes_constant\n",
     "import matplotlib.pyplot as plt\n",
     "import numpy as np\n",
     "\n",
-    "warnings.simplefilter(\"ignore\")  # Hide warnings to avoid clutter in the notebook\n",
-    "%matplotlib inline"
+    "warnings.simplefilter(\"ignore\")  # Hide warnings to avoid clutter in the notebook"
```

Comment:
It would be nicer to deal with the warning instead

`UserWarning: Converting non-nanosecond precision timedelta values to nanosecond precision`

Is the only one that shows up for me. We can either fix the root cause or add the warning filter (which can filter this specific warning) in `io/silixa.py`